mypy/build.py

```diff
@@ -2129,8 +2167,6 @@ def dispatch(sources: List[BuildSource], manager: BuildManager) -> Graph:
                       stubs_found=sum(g.path is not None and g.path.endswith('.pyi')
                                       for g in graph.values()),
                       graph_load_time=(t1 - t0),
-                      fm_cache_size=len(manager.find_module_cache.results),
-                      fm_dir_cache_size=len(manager.find_module_cache.dirs),
```

Comment:
If you refactor `lru_cache` uses away, these could be restored. Or maybe these stats could be retrieved from the lru caches?

Reply:
Yes I should add these back based on the `lru_cache.cache_info()` `currsize`.